packages/kbn-apm-types/src/es_fields/apm.ts

```diff
@@ -8,6 +8,7 @@
  */
 
 export const TIMESTAMP = 'timestamp.us';
```

Comment:
is this PR a good opportunity to change this to `TIMESTAMP_US` for better naming?

Reply:
Looks like it's referenced only in a few places. I'll rename it.